### PR TITLE
Frontend Main Page

### PR DESCRIPTION
**Pull request by jtam360**
MyMood Database Interface frontend main page
Initial layout for database interface, about us, and help page
Added Nav Bar
Added Display Table in database.html

### CODE REVIEW

File: database.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	<br>
+	
+	<!-- Database Table -->
+	<div class="container table-responsive">
+		<h1>MyMood Database Table</h1>
+		<!-- Search Bar 
+		TODO: Add more filtering options other than text -->
+		<input class="form-control" id="searchInput" type="text" placeholder="Search..">
+		<br>
+		<table class="table table-sm table-striped table-bordered table-hover id="session_table">
```
Comment (lydarren) [inline database.html]: Mismatched quotes? Attribute "id" in string "table table-sm table-striped table-bordered table-hover id=".

File: database.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	
+	<!-- jQuery library, then Popper JS, then JavaScript -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+	
+	<title>MyMood Database Interface</title>
+  </head>
+  
+  <body>
```
Comment (danqo) [inline database.html]: Let's keep the indentation consistent. It's using 2 spaces here, but tabs everywhere else.

We should decide on 1 and have all our HTML pages follow the same paradigm.
What do you think of following this: https://www.w3schools.com/html/html5_syntax.asp
I'll adjust the style for the dev guidelines / user manual later to match what we decide on now

File: aboutus.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	<!-- jQuery library, then Popper JS, then JavaScript -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+	
+	<title>MyMood Database Interface</title>
+  </head>
+  
+  <body>
+
```
Comment (danqo) [inline aboutus.html]: empty line intended?

File: database.html (hunk 1 above)
Comment (danqo) [reply database.html]: looks like it should be
<table class="table table-sm table-striped table-bordered table-hover" id="session_table">

File: database.html (hunk 2 above)
Comment (jtam360) [reply database.html]: I checked out both the links, I changed all the indentation to 2 spaces.

File: aboutus.html (hunk 3 above)
Comment (jtam360) [reply aboutus.html]: No, not intended. Fixed.

**Pull request merged**